src/gui/Src/BasicView/Disassembly.cpp

```diff
@@ -1928,7 +1928,7 @@ QString Disassembly::getAddrText(dsint cur_addr, char label[MAX_LABEL_SIZE])
         *label_ = 0;
     if(label)
         strcpy_s(label, MAX_LABEL_SIZE, label_);
-    return addrText;
+    return std::move(addrText);

```

Comment:
I'm actually setting up all the compilation tools you require (since I usually work with gcc and I haven't touched Qt in a while) to tackle a couple of bugs :P

If I see anything along the way that looks like it could be better, I will be writing it down (and I will open a PR).
I don't think there will be much though, the code looks nice from the limited amount I've seen up to now :)